### PR TITLE
Add basic spatial filtering with Cucumber tests

### PR DESCRIPTION
**Pull request by pcprince**
Initial implementation of #5

### CODE REVIEW

File: src/test/resources/net/robinjam/aes/spatialFilter.feature [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+         As a flight planner
+         I want to receive NOTAMs relating to a geographical location
+         So that I can filter out events that are not relevant to my flight plan
+
+   Scenario: Subscribe to a time period and recieve NOTAMS registered for that period
+              Given a broker is running
+              When I start a consumer with a latitude of "0.5", a longitude of "0.5" and a range of "0.1"
+              And I start a producer with xml "/delta.xml"
+              Then the consumer receives events with location "ABCD"
+              And the consumer does not receive an event with location "EFGH"
```
Comment (robinjam) [inline src/test/resources/net/robinjam/aes/spatialFilter.feature]: Should be an extra newline at the end of this document, but it's not a huge issue.

**Comment by robinjam** [comment]
Looks good to me.

File: src/test/resources/net/robinjam/aes/spatialFilter.feature (hunk 1 above)
Comment (pcprince) [inline src/test/resources/net/robinjam/aes/spatialFilter.feature]: Should I add one to the temporal filter feature file too?

Comment (robinjam) [inline src/test/resources/net/robinjam/aes/spatialFilter.feature]: Yeah, but not as part of this pull request.

**Pull request merged**